src/configs.ts

```diff
@@ -15,14 +15,48 @@ const configFilenames = [
   'eslint.config.cts',
 ]
 
+export interface ReadConfigOptions {
+  cwd: string
+  userConfigPath?: string
+  userBasePath?: string
+  /**
+   * Change current working directory to rootPath
+   * @default true
+   */
+  chdir?: boolean
+}
+
 export async function readConfig(
```

Comment:
Can you add some JSDoc for this function so it's easier to understand what exactly is being passed in?